### PR TITLE
Hook event details section up to selected timeline events.

### PR DESCRIPTION
**Pull request by kenzieschmoll**
Screen Shot 2019-11-15 at 2 45 09 PM
Also added a visualization for event details when no event is selected:
Screen Shot 2019-11-15 at 3 03 54 PM

### CODE REVIEW

File: packages/devtools_app/lib/src/timeline/flutter/event_details.dart [hunk 1]
```diff
@@ -15,25 +15,39 @@ import '../timeline_model.dart';
...
+            style: textTheme.title,
           ),
         ),
         const PaddedDivider.thin(),
         Expanded(
-          child: selectedEvent.isUiEvent
-              ? CpuProfilerView()
-              : EventSummary(selectedEvent),
+          child: selectedEvent != null
+              ? selectedEvent.isUiEvent
```
Comment (DaveShuckerow) [inline packages/devtools_app/lib/src/timeline/flutter/event_details.dart]: This is hard to understand at a glance.  Can you refactor?

File: packages/devtools_app/lib/src/timeline/flutter/timeline_screen.dart [hunk 2]
```diff
@@ -94,7 +91,7 @@ class TimelineScreenBodyState extends State<TimelineScreenBody> {
               firstChild: TimelineFlameChart(),
               // TODO(kenz): use StreamBuilder to get selected event from
```
Comment (DaveShuckerow) [inline packages/devtools_app/lib/src/timeline/flutter/timeline_screen.dart]: Is this TODO still needed now that we're just retrieving the data from the controller?

File: packages/devtools_app/lib/src/timeline/flutter/event_details.dart [hunk 3]
```diff
@@ -15,25 +15,39 @@ import '../timeline_model.dart';
...
+                : noEventSelected,
+            style: textTheme.title,
           ),
         ),
         const PaddedDivider.thin(),
         Expanded(
-          child: selectedEvent.isUiEvent
-              ? CpuProfilerView()
-              : EventSummary(selectedEvent),
+          child: selectedEvent != null
```
Comment (jacob314) [inline packages/devtools_app/lib/src/timeline/flutter/event_details.dart]: nit: not a huge fan of nested `: ?` 
Maybe move 
```dart
selectedEvent.isUiEvent
              : EventSummary(selectedEvent),	                  ? CpuProfilerView()
                  : EventSummary(selectedEvent)
```
into a helper method for clarity.

File: packages/devtools_app/lib/src/timeline/flutter/timeline_screen.dart (hunk 2 above)
Comment (kenzieschmoll) [reply packages/devtools_app/lib/src/timeline/flutter/timeline_screen.dart]: removed

File: packages/devtools_app/lib/src/timeline/flutter/event_details.dart (hunk 3 above)
Comment (kenzieschmoll) [reply packages/devtools_app/lib/src/timeline/flutter/event_details.dart]: moved into helpers

File: packages/devtools_app/lib/src/timeline/flutter/event_details.dart (hunk 1 above)
Comment (kenzieschmoll) [reply packages/devtools_app/lib/src/timeline/flutter/event_details.dart]: done. used helpers

**Pull request merged**